src/t_stream.c

```diff
@@ -1441,9 +1456,29 @@ void xrangeGenericCommand(client *c, int rev) {
     long long count = -1;
     robj *startarg = rev ? c->argv[3] : c->argv[2];
     robj *endarg = rev ? c->argv[2] : c->argv[3];
-
-    if (streamParseIDOrReply(c,startarg,&startid,0) == C_ERR) return;
-    if (streamParseIDOrReply(c,endarg,&endid,UINT64_MAX) == C_ERR) return;
+    int startex = 0, endex = 0, invalid_id = 0;
+    char *startp = startarg->ptr, *endp = endarg->ptr;
+    size_t startlen = sdslen(startarg->ptr), endlen = sdslen(endarg->ptr);
+
+    /* Check for open/close (inclusive/exclusive) range prefixes. */
```

Comment:
let's update the top comment for xrangeCommand and xrevrangeCommand

Reply:
what you did is good (top comment for xrangeGenericCommand), but i meant a top comment for xrangeCommand and xrevrangeCommand (the one that specifies the syntax).
let's have that too.

Reply:
Added :)